### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by linwei0763** [aRI0U/RandLA-Net-pytorch#57]
Hi, thank you for your work.

I am reading your code while there seems to be a mistake.

model.py line 100
        return torch.cat((
            self.mlp(concat),
            features.expand(B, -1, N, K)
        ), dim=-3)

features.expand(B, -1, N, K)
the features should query neighbours instead of  simply duplicate the values.
